prow/plugins/trigger/generic-comment.go

```diff
@@ -29,6 +31,24 @@ import (
 	"k8s.io/test-infra/prow/plugins"
 )
 
+var (
+	testHelpRe          = regexp.MustCompile(`(?m)^(/test\s*\?|/list-tests)$`)
+	emptyTestRe         = regexp.MustCompile(`(?m)^/test\s*$`)
+	retestWithTargetRe  = regexp.MustCompile(`(?m)^/retest\s+\S+$`)
+	testWithAnyTargetRe = regexp.MustCompile(`(?m)^/test\s+\S+$`)
+
+	testWithoutTargetNote = "The `/test` command needs one or more targets. "
```

Comment:
Wouldn't it make sense to add a trailing `\n` to all of them so they get their own line?

Reply:
Thank you for the review. Fixed.